client/src/Components/SearchBox.js

```diff
@@ -1,13 +1,51 @@
-import React from "react";
+import React, { useContext, useState } from "react";
+import { Item } from "../App";
 
-const SearchBox = (props) => {
+const SearchBox = () => {
+  const [Search, setSearch] = useState("");
+  const { setItemList } = useContext(Item);
+  //handleSearchSubmit
+  const handleSearchSubmit = async (e) => {
+    e.preventDefault();
+    //requesting server to fetch Search data
+    const requestOptions = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ Search: Search }),
+    };
+    const result = await (await fetch("/search", requestOptions)).json();
```

Comment:
don't use nested await. Either use await in different vars or use then-catch statements